mysql-test/suite/innodb/t/doublewrite_debug.test

```diff
@@ -103,8 +112,9 @@ set global innodb_saved_page_number_debug = 1;
 set global innodb_fil_make_page_dirty_debug = 0;
 
 set global innodb_buf_flush_list_now = 1;
-let $shutdown_timeout=0;
---source include/shutdown_mysqld.inc
+
+--let CLEANUP_IF_CHECKPOINT=drop table t1;
```

Comment:
We already defined `CLEANUP_IF_CHECKPOINT`. It is an environment variable in the client process; it will not be forgotten during a server restart.